vue-frontend/src/components/android-app-development/FaqSection.vue

```diff
@@ -0,0 +1,262 @@
+<template>
+  <div
+    class="tw-mb-[1rem] tw-bg-gradient-to-t tw-from-[#F2709C]/[80%] tw-via-[#FF9472]/[50%] tw-to-white tw-bg-gradient-[1deg] md:tw-container md:tw-bg-none"
```

Comment:
this gradient is not looking perfect as it is hiding image in mobile devices do it according to figma you can add it according to figma in config file to use all colors